UnitsNet/QuantityTypeConverter.cs

```diff
@@ -146,12 +146,13 @@ public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceT
             TAttribute? attribute = (TAttribute)context.PropertyDescriptor.Attributes[typeof(TAttribute)];
             if (attribute != null)
             {
-                QuantityType expected = default(TQuantity).Type;
-                QuantityType actual = QuantityType.Undefined;
+                // TODO
```

Comment:
TODO to address

Reply:
This still need to be fixed.

Reply:
close this duplicate feedback item (see below)

Reply:
Maybe tomorrow